src/experiments.js

```diff
@@ -124,11 +124,21 @@ export function isExperimentOnAllowUrlOverride(win, experimentId) {
  * @return {boolean}
  */
 function calcExperimentOn(win, experimentId) {
-  const cookieFlag = getExperimentIds(win).indexOf(experimentId) != -1;
+  const experiments = getExperimentIds(win);
+
+  // Disabling cookie flag.
+  const disableFlag = experiments.indexOf('-' + experimentId) != -1;
```

Comment:
Using `AMP.toggleExperiment('xxx', false)` can manually opt-out for the current page view. I guess that's not enough for your use case? You want it to be persistent?